src/Utilities/Helpers.js

```diff
@@ -563,3 +564,29 @@ export const systemsColumnsMerger = defaultColumns => {
 
     return [...nameAndTag, ...systemsListColumns, lastSeen[0]];
 };
+
+export const activateFeatureFlags = (location) => {
+    let environment = {};
+
+    environments.forEach(env => {
+        if (window.location.href.includes(env.key)) {
+            environment.name = env.name;
+        }
+    });
+
+    const enabledFeatures = featureFlags.filter(flag => {
+        return location.search.includes(flag.urlKey) || environment.name === 'dev' || 'stage';
```

Comment:
I have a few suggestions, but let me know if I'm misunderstanding how the feature flag works.

I think you mean for this line to read: `return location.search.includes(flag.urlKey) || environment.name === 'dev' || environment.name === 'stage';` In my testing, this line of code would return '`stage` if the first 2 were false, even though I was in prod.

Also, I noticed that if I added `patch_set=enabled` to the url like so: `https://stage.foo.redhat.com:1337/insights/patch/systems?page=1&perPage=20&sort=-last_upload&filter[stale]=in%3Atrue%2Cfalse&patch_set=enabled`, it would eventually reset the url to not include it. At first it would take in the parameter, but something in the code is resetting the location and then it sets it to false.

Reply:
Yeap, I did a mistake while writing this condition :)

What I wanted to achive was what you have mentioned: 
`return location.search.includes(flag.urlKey) || environment.name === 'dev' || environment.name === 'stage';`